app/qml/components/MMLinkedFeaturesDrawer.qml

```diff
@@ -0,0 +1,168 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+import "../inputs"
+
+Drawer {
```

Comment:
Quick question about drawers, can there be multiple of them opened at the same time? This page can be opened multiple times at the same time

Reply:
Yes, multiple drawers can be opened at the same time.

Reply:
https://github.com/MerginMaps/mobile/assets/72412563/9ebb4d50-93b3-419f-ae11-64e96df5b829

Reply:
Tested also on mobile

Reply:
Great! :) Thanks